src/components/Blocking/ReportDomain.js

```diff
@@ -79,7 +80,7 @@ const ReportDomain = React.forwardRef((props, ref) => {
               <Text style={styles.btnSkipText}>
                 Skip & just block this account
               </Text>
-              <IconFA5 name="chevron-right" size={17} color={'#000'} />
+              <IconFA5 name="chevron-right" size={17} color={COLORS.black} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from hard-coded color to using a centralized theme for colors is a good practice. It improves maintainability and consistency across the application.
```diff
- <IconFA5 name="chevron-right" size={17} color={'#000'} />
+ <IconFA5 name="chevron-right" size={17} color={COLORS.black} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
@agit26milan masih ada `import {colors} ...` bisa dipastikan disemua file juga mas, karena rasanya masih banyak yang dobel import yang `colors`